wherehows-web/app/components/get-yield.ts

```diff
@@ -0,0 +1,7 @@
+import Component from '@ember/component';
+
+/**
+ * Small component that will simply yield get a property of an object
```

Comment:
This sounds more like it would clutter hbs. Not sure what the advantage is?

Reply:
So, for maps type of things it is useful. This usually happens when doing a redux approach, and you don't want to mutate the core object (this will cause rerender on UI of the whole thing), so you use a map to define those things that will mutate. In this case this is the count and selected.

Reply:
Ah I see. Disregard the follow up in the second half of the review then lol